AGSnapshotHelperTests/AGPreferredContentSizeCategoryMockerTests.m

```diff
@@ -0,0 +1,66 @@
+//
+//  AGPreferredContentSizeCategoryMockerTests.m
+//  AGSnapshotHelper
+//
+//  Created by Adam Grzegorowski on 24/11/15.
+//  Copyright (c) 2015 allegro. All rights reserved.
+//
+
+#import "AGPreferredContentSizeCategoryMocker.h"
+
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
+
+@interface AGPreferredContentSizeCategoryMockerTests : XCTestCase
+
+@end
+
+@implementation AGPreferredContentSizeCategoryMockerTests
+
+- (void)testPreferredContentSizeCategoryShouldBeEqualToSetValueAfterStartingMocking {
```

Comment:
Maybe testPreferredContentSizeCategoryShouldBeEqualToSetValueAfterStarMocking? Up to you.